doc/en/user/source/configuration/crshandling/coordtransforms.rst

```diff
@@ -71,9 +71,9 @@ NTv2
    4230,4326,SPED2ETV2.gsb,OGP
    4230,4326,sped2et.gsb,OGP
    4258,4275,rgf93_ntf.gsb,OGP
-   4267,4269,NTv2_0.gsb,http://www.geod.nrcan.gc.ca/products/html-public/GSDapps/English/NTv2_Fact_Sheet.html
+   4267,4269,ntv2_0.gsb,Natural Resources Canada; https://open.canada.ca/data/en/dataset/b3534942-31ea-59cf-bcc3-f8dc4875081a
```

Comment:
I've followed the link, the file that gets downloaded is a NTV2_0.GSB ... all uppercase?

Reply:
Yes, the source file is from 1995 DOS era, thus is uppercase. But having checked many sources across the internet, I don't see anyone using this file as uppercase. Usually mixed or lowercase. PROJ and GDAL have been using the lowercase name for quite a while.

Reply:
PROJ has been hard-coded to use the lowercase name since version 4.4.7 (released 2003-03-31, over 15 years ago), and GDAL for probably a similar time-frame (I haven't checked).
And I mentioned other projects use an impressively wide range of mixed case including `Ntv2_0.gsb` ([Esri](http://help.arcgis.com/en/geodatabase/10.0/sdk/arcsde/concepts/geometry/coordref/coordsys/geographic/usinggrid_gt.htm)), `NTV2_0.gsb` ([Safe](https://docs.safe.com/fme/html/FME_Desktop_Documentation/FME_Desktop_Admin_Guide/FMEInstallation/Installing_Grid_Shift_Files.htm), [Autodesk](https://knowledge.autodesk.com/support/autocad-map-3d/learn-explore/caas/CloudHelp/cloudhelp/2017/ENU/MAP3D-Use/files/GUID-C6B3BE91-1BC3-4679-87AE-64DD4A36FBD4-htm.html)), `ntv2_0.GSB` ([DotSpatial](https://github.com/DotSpatial/DotSpatial/blob/master/Source/DotSpatial.Projections.Tests/GeogTransformGrids/ntv2_0.GSB)). Having searched this file's use, I haven't yet seen a modern software project that uses the original all-caps file name (this file was primarily aimed for DOS developers, [see archived info](https://web.archive.org/web/20010227084518/http://cgis.geod.nrcan.gc.ca/products/html-public/GSDapps/English/NTv2_Fact_Sheet.html)).

I've also just checked the EPSG Registry, and as of version 9.4 it still uses `NTv2_0.gsb`. The choice of names is yours to make :smile:.

Reply:
Can you please update the pull request so that the file name matches the one downloaded, without requiring renames?